File: python/35d_1676B_EqualCandies.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        arr = input().split()
        for i in range(n):
            arr[i] = int(arr[i])
        arr.sort()
        cnt = 0
        for i in arr:
            cnt += i - arr[0]
        ans.append(cnt)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()

File: cpp/35d_1676B_EqualCandies.cpp
#include<bits/stdc++.h>

int main(){
    int t, n, temp, cnt;
    std::vector<int> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n;
        std::vector<int> arr;
        for (int j = 0; j < n; j++){
            std::cin>>temp;
            arr.push_back(temp);
        }
        sort(arr.begin(), arr.end());
        cnt = 0;
        for(int j = 1; j < n; j++){
            cnt += arr[j] - arr[0];
        }
        ans.push_back(cnt);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}